lib/services/product_service.dart

```diff
@@ -46,6 +47,25 @@ class ProductServices {
       description: data["description"],
     );
   }
-}
 
+  var titleController = TextEditingController();
```

Comment:
You shouldn’t use flutter controllers in services.
You should accept the value as parameters